### PR TITLE
Animation Cleanup: En_M*

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/overlays/actors/ovl_En_Bubble/z_en_bubble.c [hunk 1]
```diff
@@ -333,7 +333,7 @@ void EnBubble_Init(Actor* thisx, PlayState* play) {
     Collider_InitJntSph(play, &this->colliderSphere);
     Collider_SetJntSph(play, &this->colliderSphere, &this->actor, &sJntSphInit, this->colliderElements);
     CollisionCheck_SetInfo2(&this->actor.colChkInfo, DamageTable_Get(9), &sColChkInfoInit);
-    this->actor.hintId = 0x16;
+    this->actor.hintId = TATL_HINT_ID_IGOS_DU_IKANA;
```
Comment (isghj5) [inline src/overlays/actors/ovl_En_Bubble/z_en_bubble.c:336]:
```suggestion
    //! @bug: hint Id not correctly migrated from OOT `NAVI_ENEMY_SHABOM`
    this->actor.hintId = TATL_HINT_ID_IGOS_DU_IKANA;
```
Can confirm, this is the wrong hint. Can we add a bug documentation for it?
Screenshot_2023-11-14_18-06-52

**Review by jpburnett — APPROVED** [review]
Noice

File: src/overlays/actors/ovl_En_Mm3/z_en_mm3.c [hunk 2]
```diff
@@ -63,15 +62,27 @@ static ColliderCylinderInit sCylinderInit = {
...
+
+static AnimationInfo sAnimationInfo[ENMM3_ANIM_MAX] = {
+    { &object_mm_Anim_002238, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },   // ENMM3_ANIM_0
+    { &object_mm_Anim_00A4E0, -1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -7.0f },  // ENMM3_ANIM_1
+    { &object_mm_Anim_00C640, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },   // ENMM3_ANIM_2
+    { &object_mm_Anim_00A4E0, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -7.0f },   // ENMM3_ANIM_3
+    { &object_mm_Anim_000468, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -7.0f },   // ENMM3_ANIM_4
+    { &object_mm_Anim_00CD90, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -12.0f },  // ENMM3_ANIM_5
+    { &object_mm_Anim_00DA50, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -12.0f },  // ENMM3_ANIM_6
+    { &object_mm_Anim_00DA50, 1.0f, 0.0f, 10.0f, ANIMMODE_ONCE, -10.0f }, // ENMM3_ANIM_7
```
Comment (isghj5) [inline src/overlays/actors/ovl_En_Mm3/z_en_mm3.c:85]: er, isn't this going to clash with the postman pr?

File: src/overlays/actors/ovl_En_Bubble/z_en_bubble.c (hunk 1 above)
Comment (engineer124) [reply src/overlays/actors/ovl_En_Bubble/z_en_bubble.c:336]: It seems like this same bug is also present in `z_en_elfbub.c` (stray fairies in bubbles), I'll mark that bug as well

File: src/overlays/actors/ovl_En_Mm3/z_en_mm3.c (hunk 2 above)
Comment (engineer124) [reply src/overlays/actors/ovl_En_Mm3/z_en_mm3.c:85]: Yeah, good point. That PR just focuses on the object, so this still has new parts. But it's also done by a new contributor. I can help guide them merge the conflict if need be, I'll keep a close eye on that PR

**Review by hensldm — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**